docs/UserGuide.md

```diff
@@ -160,21 +160,51 @@ Lists residents whose ART collection or FET tests are either:
-
-Format: `deadline k/KEYWORD d1/DATE1 d2/DATE2` or `deadline k/LATE_KEYWORD d1/DATE1`
-
-* A resident's fet or collection is due one week after their last fet date or last collection date. 
-  * For example, if a resident's last fet date is on a friday, `15-10-2021`, then the resident's fet deadline is on the following friday which is `22-10-2021`
-* The number of days a resident is considered late is calculated from `a day after` their deadline to the `current date`, both inclusive. 
-  * For example, if a resident's last fet date is `15-10-2021`, then the fet deadline will be `22-10-2021`, if the current date is `25-10-2021`, then the number of days the resident is late for fet is `3 days`.
-* Normal keywords are `f` for fet and `c` for collection
-* Late keywords are `lf` for late fet and `lc` for late collection
-* When a normal keyword is given, both `DATE1` and `DATE2` have to be inputted
-* The given `DATE2` must be a date later than the given `DATE1`
-* `DATE1` is the start date and `DATE2` is the last date inclusive
-* When a late keyword is given, only date1 should be given
-* Anyone whose fet and collection is due before but not on `DATE1` is outputted
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+* There is a one week deadline for the test kit collection and fet test, therefore due date refers to one week after the last recorded date
+
+</div>
+
+| Differences | Normal Keyword | Late Keyword |
+|-------------|--------|------|
+| Format | `deadline k/KEYWORD d1/DATE1 d2/DATE2` | `deadline k/LATE_KEYWORD d1/DATE1`|
+| Keyword | `f` or `c` | `lf` or `lc` |
+| Input Date | Both `DATE1` and `DATE2` have to be inputted | Only `DATE1` should be inputted |
+| Command Example | `deadline k/f d1/10-10-2021 d2/12-10-2021` | `deadline k/lf d1/11-10-2021` |
+| Usage | List residents whose deadline lie within the range of two given dates, inclusive | List residents whose deadline is due before a given date
+| Usage Example | A resident's fet or collection is due one week after their `last fet date` or `last collection date`. <br> For example, if a resident's last fet date is on a friday, `15-10-2021`, then the resident's fet deadline is on the following friday which is `22-10-2021` | The number of days a resident is considered late is calculated from `a day after` their deadline to the `current date`, both inclusive.<br> For example, if a resident's last fet date is `15-10-2021`, then the fet deadline will be `22-10-2021`, if the current date is `25-10-2021`, then the number of days the resident is late for fet is `3 days`.
+| Note | The given `DATE2` must be a date later than the given `DATE1`<br>`DATE1` is the start date and `DATE2` is the last date inclusive | Anyone whose fet and collection is due before but not on `DATE1` is outputted
+
+Here's a step by step guide for `Normal Keyword`:<br>
+1. Navigate to the `Resident` Tab
+   ![Step1](images/logic/commands/deadlinecommand/step1.png)
```

Comment:
What do you think of removing the navigate screenshot for each step-by-step? Seems like it takes a lot of unnecessary space and one bug mentioned in the website is: `Too many visuals and duplicated information`

Reply:
I agree